frontend/src/lib/lemon-ui/LemonRadio/LemonRadio.tsx

```diff
@@ -0,0 +1,46 @@
+import clsx from 'clsx'
+
+import { IconCheckmark, IconRadioButtonUnchecked } from '../icons'
```

Comment:
`@posthog/icons` are the way forward, so it would be ideal to avoid using the old Material icons

Reply:
A checkbox is just a rounded box, so we should be able to do this super cleanly with just HTML